packages/api/tests/entities/Module.test.ts

```diff
@@ -67,14 +74,28 @@ describe('Module entity', () => {
   describe('Mutation: updateModule', () => {
     beforeEach(async () => {
       await setup();
-      path = await TestClient.createPath(pathInput, characterInput);
+      path = await TestClient.createPath(pathInput);
     });
 
     it('should update a module successfully', async () => {
-      expect.assertions(6);
-      const modulePrev = await TestClient.createModule({ ...moduleInput, name: 'previous', pathId: path.id });
-      const module = await TestClient.createModule({ ...moduleInput, pathId: path.id });
-      const path2 = await TestClient.createPath({ ...pathInput, name: 'path 2' }, {name: 'name 2', displayName: 'display name 2'});
+      expect.assertions(2);
+
+      const modulePrev = await TestClient.createModule({
+        ...moduleInput,
+        name: 'previous',
```

Comment:
I think you are still using prettier and makes de reviews harder to review having to pay attention to changes that aren't actually changes.

Reply:
I thought the vscode settings you gave me would have helped with this issue. Since then, whenever I run yarn lint I see no issues. I've since uninstalled prettier. Not sure how that will help. I will look for a way to toggle it on and off when working on this project.